settings.gradle.kts

```diff
@@ -1,7 +1,7 @@
 buildscript { repositories { gradlePluginPortal() } }
 
 plugins {
-    id("net.vivin.gradle-semantic-build-versioning") version "4.0.0"
+    id("io.alcide.gradle-semantic-build-versioning") version "4.2.2"
```

Comment:
I was having trouble running Gradle locally. This fixed the issue and is the spiritual successor to the unmaintained original plugin we use currently.

Reply:
I don't have experience with this plugin, but since it is now a fork of a fork we may encounter future ownership issues again at some point.